apps/aeutils/test/data/epoch_full.yaml

```diff
@@ -50,6 +52,16 @@ sync:
     gossip_allowed_height_from_top: 5
     # Time (milliseconds) between logging info about connected peers
     log_peer_connection_count_interval: 5000
+    # Peer pool
+    peer_pool:
+      # Probability of selecting a peer from the verified pool
+      select_verified_peer_probability: 0.5
+      # Time (milliseconds) without a peer being updated after which it gets removed
+      max_update_lapse: 2592000000
```

Comment:
If this PR is not intended to change defaults, this shall be 2,160,000,000 instead